src/main/java/com/mathworks/ci/RunMatlabTestsStep.java

```diff
@@ -93,6 +94,14 @@ public void setModelCoverageCobertura(String modelCoverageCobertura) {
         this.modelCoverageCobertura = modelCoverageCobertura;
     }
 
+    public List<String> getSourceFolder() {
```

Comment:
I did not see the config.jelly changes for this file.

Reply:
Made appropriate changes